roles/provision-keycloak-apb/tasks/provision-keycloak.yml

```diff
@@ -110,15 +110,20 @@
   shell: "oc get routes keycloak -n '{{ namespace }}' | grep -v NAME | awk '{print $2}'"
   register: keycloak_route
 
-- copy:
+- name: "Copy keycloak-metrics-spi jar into temp directory"
+  copy:
     src: keycloak-metrics-spi-1.0-SNAPSHOT.jar
     dest: /tmp/keycloak-metrics-spi-1.0-SNAPSHOT.jar
 
-- shell: oc get pods --namespace {{ namespace }} --selector="deploymentconfig=keycloak" | awk 'NR > 1 {print $1}'
-  register: get_pods
+- name: Get the name of the keycloak pod
+  shell: oc get pods --namespace {{ namespace }} --selector="deploymentconfig=keycloak" | awk 'NR > 1 {print $1}' | grep -v deploy
```

Comment:
As an alternative to awk and grep you could also use a jsonpath query. For example like this: https://github.com/aerogearcatalog/keycloak-apb/blob/master/roles/provision-keycloak-apb/tasks/label-serviceinstance.yml#L2

Reply:
Awesome. Good suggestion, I'll replace it then.